build.sh

```diff
@@ -92,6 +94,9 @@ fi
 if hasArg benchmarks; then
     BENCHMARKS="ON"
 fi
+if hasArg kafka_tests; then
```

Comment:
Do you want these tests to run during CI? If so, you should update in `ci/gpu/build.sh` to use this flag and run the tests.

Reply:
I actually don't want them to run. They are more integration test focused and require a running Kafka instance to pass so I decided to disable them by default for CI.